samples/textcompletion/csharp-ooproc/host.json

```diff
@@ -0,0 +1,12 @@
+{
+    "version": "2.0",
+    "logging": {
```

Comment:
It might be worth making this host.json file match the in-proc version. It seems like this one just has default settings.